openfisca_core/periods/instant_.py

```diff
@@ -195,6 +209,10 @@ def offset(self, offset, unit):
             return self.__class__((date.year, date.month, date.day))
         return None
 
-    @property
-    def year(self):
-        return self[0]
+    @classmethod
+    def eternity(cls) -> t.Instant:
```

Comment:
Why do we need this ?

Reply:
Creating eternity instants was repeated through four times. I thought it made it easier to test. That makes it definitely a minor bump, thanks for seeing it. I'll update the PR.

Reply:
Add it to the changelog.